pkg/services/system_intakes.go

```diff
@@ -356,6 +357,11 @@ func NewArchiveSystemIntake(
 			}
 		}
 
+		err = sendWithdrawEmail(intake.ProjectName.String)
+		if err != nil {
+			return err
```

Comment:
Non-blocking: My question is about the error handling here. If we've done all the previous things in this service, and we get to the email part, and somewhere/somehow we have a problem sending the email, is that really the end-users' concern? My initial gut reaction would be to just log the error for the operators of the system to be informed about, but to succeed the overall request.

Reply:
ooh, that's a good point. If we errored here, the requester wouldn't be able to re-submit it (since it'd already be submitted in the db)